CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# How To Contribute
+
```

Comment:
Karan also pointed me to https://wiki.lfnetworking.org/display/L3AF/Getting+Started+with+L3AF which contains a bunch of information not in this document.  We should either incorporate it or point to that page.